README

```diff
@@ -1548,16 +1549,30 @@ To display the DOSBox status window:
 =====================================
 
 The configuration file is automatically created the first time you run DOSBox.
-The file can be found in:
-   (Windows)  "Start/WinLogo Menu"->"All Programs"->"DOSBox-0.74-2"->Options
-   (Linux)    ~/.dosbox/dosbox-0.74-2.conf
-   (Mac OS X) "~/Library/Preferences/DOSBox 0.74-2 Preferences"
+
+The default directories storing config file are:
+
+   (Windows) C:\Users\<username>\AppData\Local\DOSBox\
+   (Linux)   ~/.config/dosbox/
+   (macOS)   ~/Library/Preferences/DOSBox/
+
+Linux users:
+    The configuration's parent-directory can be customized by setting
+    the XDG_DATA_HOME environment variable, per the XDG Base Directory
+    Specification. If set, DOSBox will use $XDG_CONFIG_HOME/dosbox/ for
+    its configuration path.
+
+Windows users:
+    In Windows Vista or newer, the configuration file won't work correctly
+    if it is located in "Windows" or "Program Files" folder or their subfolders,
+    or directly on C:\, so the best place for storing extra configuration files
+    is next to the default config directory or for example: C:\oldgames
+
```

Comment:
Helpful clarification; head-off gotchas before people hit them.

Reply:
This description was there in the end of configuration file section, I moved it up, so platform-specific descriptions are next to each other.